.github/workflows/ci.yaml

```diff
@@ -0,0 +1,67 @@
+name: CI
+
+on:
+  # run once a day at noon UTC
+  schedule:
+    - cron: "0 12 * * *"
+  push:
+    branches:
+      - "master"
+  pull_request:
+    branches:
+      - "master"
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: 
+          - ubuntu-latest
+         #- macOS-latest 
+        python-version: 
+          - 3.7
+          - 3.8
+          - 3.9
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+        architecture: x64
+
+    - name: Environment Information
+      shell: bash -l {0}
+      run: |
+        python --version
+
+    - name: Install pytest, pytest plugins
+      shell: bash -l {0}
+      run: |
+        python -m pip install pytest pytest-cov pytest-pep8
+
+    - name: Install alchemtest
+      shell: bash -l {0}
+      run: |
+        python -m pip install --only-binary=numpy numpy # Otherwise this would take ages
+        python -m pip install https://github.com/alchemistry/alchemtest/archive/master.zip
+
+    - name: Install package
+      shell: bash -l {0}
+      run: |
+        python -m pip install -e .
```

Comment:
Do we really want a developer installation or is better to do a standard installation as this also checks that everything is copied appropriately? I don't remember the rationale for the developer installation, though.

Reply:
The rationale is that you're testing the same `alchemlyb` as the source files, but in this case doesn't matter since it's ephemeral anyway. I'll remove the `-e`.